### PR TITLE
Kurdish translation

### PR DESCRIPTION
**Pull request by ahalimkara**
#9027

### CODE REVIEW

**Comment by nikogu** [comment]
@ahalimkara Please fixed the ci https://travis-ci.org/ant-design/ant-design/jobs/331772954#L5335

**Comment by ahalimkara** [comment]
@nikogu fixed

**Review by yesmeck — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "omit.js": "^1.0.0",
     "prop-types": "^15.5.7",
     "rc-animate": "^2.4.1",
-    "rc-calendar": "~9.4.0",
+    "rc-calendar": "~9.5.0",
```
Comment (nikogu) [inline package.json]: it's a irrelevant change ? and the `rc-pagination`

~9.4.0: >=9.4.0 && < 9.5.0

Comment (ahalimkara) [reply package.json]: I have added Kurdish translation to rc-calender and rc-pagination. Please check this https://github.com/ant-design/ant-design/pull/9027#issuecomment-359361333 and this https://github.com/ant-design/ant-design/pull/9015#discussion_r162527232

**Pull request merged**